src/main/java/com/mongodb/hibernate/jdbc/MongoConnectionProvider.java

```diff
@@ -55,7 +55,9 @@ public final class MongoConnectionProvider implements ConnectionProvider, Stoppa
     @Serial
     private static final long serialVersionUID = 1L;
 
+    /** The StandardServiceRegistryScopedState */
```

Comment:
TIL - that in java 17 - all private variables in `Serializable` classes require javadoc.